### PR TITLE
Hit the API

### PR DESCRIPTION
**Pull request by Christianib003**
In this milestone, I added:
- The full functionality of the Leaderboard app using API.
- Changes the display of the results to table.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import './style.css';
+import Application from './c-modules.js/app.js';
+
+const application = new Application();
+application.initApp();
```
Comment (FrederickMih) [inline src/index.js]: - [ ] Your project compiles with some issues. Screenshot below:


Screenshot from 2022-05-13 11-48-42
:

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        <h2>Add Your Score</h2>
+      </div>
+      <p class="success-message"></p>
+      <div class="section-body end">
+        <form id="c-form">
+          <label for="name">Name</label>
+          <input id="name" class="userdetails" type="text" minlength="2" placeholder="Your name" required>
+          <label for="Score">Score</label>
+          <input id="Score" class="userdetails" type="number" min="0" max="100" placeholder="Your score" required>
+          <button type="submit">Submit</button>
```
Comment (FrederickMih) [inline src/index.html]: - [ ] When I click on the submit button, I got this error: 


Screenshot from 2022-05-13 11-55-22

Please fix it :pray:

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Christianib003 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



# To Highlights
 
- Linters are set up correctly :heavy_check_mark: 
- PR has a good title and summary :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+import './styles/style.css';
+import './modules/app.js';
```
Comment (ahmetbozaci) [inline src/index.js:2]: - [ ] As the last code reviewer said please fix the compiled error.

image

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+                <button id="refresh" type="button">Refresh</button>
+            </div>
+            <div class="recent-scores"></div>
+        </div>
+        <div class="add-score">
+            <h2>Add your score</h2>
+            <form action="#">
+                <input required type="text" name="name" id="userName" placeholder="Your Name">
+                <input required type="number" name="name" id="userScore" placeholder="Your Score">
+                <button id="submitScore" type="submit">Submit</button>
```
Comment (ahmetbozaci) [inline src/index.html:24]: - [ ] As the last code reviewer said you should fix the error when clicking on the submit button.

image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


@Christianib003  ✋🏼 
#### Great work on making the changes requested by a previous reviewer 👏 
#### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
___
### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: src/index.html (hunk 4 above)
Comment (iKrystyan) [reply src/index.html:24]: Hi @ahmetbozaci  
Thanks for the review, but I can't see any of those errors these errors in my console.
image

Comment (iKrystyan) [reply src/index.html:24]: If you have any suggestions, I would appreciate your help.

Comment (ahmetbozaci) [reply src/index.html:24]: @Christianib003 Answered in Slack.

**Review by KDlamini — APPROVED** [review]
### Hi @Christianib003 🙋‍♀️,  

Great job! 
You have made a great effort in implementing all the required changes. 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

# Status: Approved ✔️ 

simon

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**